### PR TITLE
feat: io support http status & add mvp controller validator

### PR DESCRIPTION
**Pull request by PainterPuppets**
- `kuai/io` support http status 
 -  `mvp-dapp` controller add some params validate

### CODE REVIEW

**Comment by Keith-CY** [comment]
Please assign reviewers if this PR is ready

**Comment by Keith-CY** [comment]
How is it going

File: packages/samples/mvp-dapp/src/chain-source.ts [hunk 1]
```diff
@@ -13,7 +13,7 @@ export class NervosChainSource implements ChainSource {
 
   getCurrentEpoch = () => this.#rpc.getCurrentEpoch()
 
-  getBlock = (blockNumber: string) => this.#rpc.getBlock(blockNumber)
+  getBlock = (blockNumber: string) => this.#rpc.getBlockByNumber(blockNumber)
```
Comment (PainterPuppets) [inline packages/samples/mvp-dapp/src/chain-source.ts:16]: Here the use of `getBlock` will thorw an error, so changed to `getBlockByNumber`

File: packages/samples/mvp-dapp/src/main.ts [hunk 2]
```diff
@@ -29,7 +29,7 @@ async function bootstrap() {
 
   const koaRouterAdapter = new KoaRouterAdapter(cor)
 
-  app.use(koaRouterAdapter.routes()).use(koaRouterAdapter.allowedMethods())
+  app.use(koaRouterAdapter.routes())
```
Comment (PainterPuppets) [inline packages/samples/mvp-dapp/src/main.ts:32]: `Methods` is managed by `kuaiRouter`, so there is no need to use `allowedMethods`.

**Review by yanguoyu — APPROVED** [review]

**Review by Daryl-L — APPROVED** [review]

**Pull request merged**